corehq/messaging/smsbackends/connectid/backend.py

```diff
@@ -0,0 +1,47 @@
+import base64
+import requests
+from Crypto.Cipher import AES
+from uuid import uuid4
+
+from django.conf import settings
+
+from corehq.apps.users.models import ConnectIDUserLink, CouchUser
+
+class ConnectBackend:
+    couch_id = "connectid"
+
+    def send(self, message):
+        user = CouchUser.get_by_user_id(message.couch_recipient).get_django_user()
+        user_link = ConnectIDUserLink.objects.get(commcare_user=user)
+        key = base64.b64decode(user_link.connectidmessagingkey_set.first().key)
+        cipher = AES.new(key, AES.MODE_GCM)
+        data, tag = cipher.encrypt_and_digest(message.text.encode("utf-8"))
+        content = {
```

Comment:
The encryption util spits out binary data, which we need to encode as base64 to get text for a json dict, but that returns a `bytes` object (`b'my text'`), rather than a python string we need for the request.